src/ledgerwriter/src/main/java/anthos/samples/financedemo/ledgerwriter/LedgerWriterController.java

```diff
@@ -59,10 +57,17 @@
 
 
     public static final String READINESS_CODE = "ok";
+    public static final String UNAUTHORIZED_CODE = "not authorized";
+    public static final String CLAIM = "acct";
+    public static final String
+            EXCEPTION_MESSAGE_WHEN_AUTHORIZATION_HEADER_NULL =
+            "HTTP request 'Authorization' header is null";
+    public static final String EXCEPTION_MESSAGE_INSUFFICIENT_BALANCE =
+            "insufficient balance";
     // account ids should be 10 digits between 0 and 9
-    private static final Pattern ACCT_REGEX = Pattern.compile("^[0-9]{10}$");
+    public static final Pattern ACCT_REGEX = Pattern.compile("^[0-9]{10}$");
     // route numbers should be 9 digits between 0 and 9
-    private static final Pattern ROUTE_REGEX = Pattern.compile("^[0-9]{9}$");
+    public static final Pattern ROUTE_REGEX = Pattern.compile("^[0-9]{9}$");
```

Comment:
These regex patterns should probably be moved to the new validation class, since that's where they're used

Reply:
You're right! That's the only place they were used. I moved them